### ISSUE TITLE
Embed with sidebar hidden

### ISSUE THREAD

**Issue by VonSub** [docsifyjs/docsify#806]
Trying to embed without the full sidebar showing, but just the hamburger toggle button.
Is that possible?

**Comment by timaschew** [comment]
You want to hide the sidebar initially (on page load)?

**Comment by VonSub** [comment]
@timaschew yes, that is indeed the idea. So people can find it, but initially will read included information.

**Comment by trusktr** [comment]
We can achieve this as a non-breaking change, behind a new option. Adding to the 4.x project board for tracking.
